web/modules/custom/portland/modules/portland_zendesk/src/Plugin/WebformHandler/ZendeskHandler.php

```diff
@@ -567,6 +567,11 @@ public function submitTicket($submission_fields, $configuration, $webform_submis
     $request['collaborators'] = preg_split("/[^a-z0-9_\-@\.']+/i", $request['collaborators'] );
     if (!empty($request['ticket_form_id'])) $request['ticket_form_id'] = $this->configuration['ticket_form_id'];
 
+    // use anonymous if no requester set
```

Comment:
Uses the default anonymous email address if one is not required by the parent form. Zendesk requires an email address.